__tests__/integration.test.js

```diff
@@ -0,0 +1,30 @@
+const app = require("../app");
+const request = require("supertest");
+const seed  = require("../db/seeds/seed")
+const db = require("../db/connection")
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(()=> {
+    db.end()
+})
+
+describe('getTopics', () => {
+    it('should return a 200 status code', () => {
+        return request(app).get("/api/topics").expect(200)
+    });
+    it('should return an array of topic objects, each of which should have the following properties: slug, description) ', () => {
+        return request(app).get("/api/topics").then((res) => {
+            expect(Array.isArray(res.body.topics)).toBe(true)
```

Comment:
Don't need to explicitly check for array here as it's implied by us being able to use a forEach on line 23